finagle-zipkin-core/src/main/scala/com/twitter/finagle/zipkin/core/Annotation.scala

```diff
@@ -1,4 +1,4 @@
-package com.twitter.finagle.zipkin.thrift
+package com.twitter.finagle.zipkin.core
```

Comment:
moving the package is a breaking change. i think its probably worth it, but please note these in the "Breaking" section of `CHANGES`.